assets/src/edit-story/components/form/dropDown/index.js

```diff
@@ -36,13 +36,13 @@ import {
 } from '../../../utils/keyboardOnlyOutline';
 import { Dropdown as DropdownIcon } from '../../../icons';
 import Popup, { Placement } from '../../popup';
-import DropDownList from './list';
+import DropDownList, { DEFAULT_MIN_WIDTH } from './list';
 
 /* same min-width as ListContainer */
 const DropDownContainer = styled.div`
   display: flex;
   flex-direction: column;
-  min-width: 160px;
+  min-width: ${({ minWidth }) => minWidth || DEFAULT_MIN_WIDTH}px;
```

Comment:
I do wonder if `flex: 1` here is the better option. Visually that seems to make it fill the container correctly without having to resort to numeric metrics. I'd imagine that most of the dropdowns are inside containers so I can't see this having ripple effects. Alternatively we could have a prop that `flex: 1` like `flexContainer` or something.

Reply:
Seems like it works! 🎉